extensions/amp-sticky-ad/0.1/amp-sticky-ad.js

```diff
@@ -38,21 +39,25 @@ class AmpStickyAd extends AMP.BaseElement {
       dev.warn(TAG, `TAG ${TAG} disabled`);
       return;
     }
-
+    toggle(this.element, true);
```

Comment:
This is needed because the `NODISPLAY` layout? Why don't we change it to `CONTAINER`?